pangea-sdk/pangea/pangea.go

```diff
@@ -161,6 +164,14 @@ func NewClient(service string, baseCfg *Config, additionalConfigs ...*Config) *C
 	cfg := baseCfg.Copy()
 	cfg.MergeIn(additionalConfigs...)
 
+	if len(cfg.BaseURLTemplate) == 0 && len(cfg.Domain) == 0 {
```

Comment:
Shouldn't we require at least one? Not sure that assuming `aws.us.` is correct.

Reply:
Why not default to `aws.us.pangea.cloud`? I would bet that's what 99% of users want anyways.